src/components/WizardFooter/WizardFooter.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { ChevronLeft } from '@amsterdam/asc-assets';
+import {
+	CancelButtonStyle,
+	PreviousButtonStyle,
+	ButtonPanelStyle,
+	LeftActionStyle,
+	RightActionStyle,
+	WizardButton,
+} from './WizardFooterStyles';
+
+export default class WizardFooter extends Component<{
+	cancelLabel: string;
```

Comment:
I think this props interface is a bit much. We only need red next/save buttons so I think we can simplify it a bit. Here's what I would make of it:
```typescript
type ButtonConfig = {
	disabled?: boolean;
	label?: string;
	onClick: (evt: React.MouseEvent<HTMLButtonElement>) => void;
};

type Props = {
	cancel?: ButtonConfig;
	next?: ButtonConfig;
	previous?: ButtonConfig;
	save?: ButtonConfig;
};

// Custom hook
function useButtonConfig(
	config: ButtonConfig | undefined,
	defaultLabel: string,
): { visible: boolean; label: string; disabled: boolean; onClick?: ButtonConfig['onClick'] } {
	return {
		visible: config !== undefined,
		label: config?.label ?? defaultLabel,
		disabled: config?.disabled ?? false,
		onClick: config?.onClick,
	};
}

const WizardFooterExample: React.FC<Props> = ({ cancel, next, previous, save }) => {
	const { visible: cancelEnabled, disabled: cancelDisabled, onClick: onCancelClick } = useButtonConfig(
		cancel,
		'Annuleren',
	);
	const { visible: nextEnabled, disabled: nextDisabled, onClick: onNextClick } = useButtonConfig(next, 'Volgende');
	const { visible: previousVisible, disabled: previousDisabled, onClick: onPreviousClick } = useButtonConfig(
		previous,
		'Vorige',
	);
	const { visible: saveVisible, disabled: saveDisabled, onClick: onSaveClick } = useButtonConfig(save, 'Opslaan');

	return <></>;
};
```

Reply:
implemented